widgetssdk/src/main/java/com/glia/widgets/di/Dependencies.java

```diff
@@ -211,9 +222,18 @@ private static void initApplicationLifecycleObserver(
     }
 
     private static void initAudioControlManager(
-            AudioControlManager audioControlManager,
-            OnAudioStartedUseCase onAudioStartedUseCase
+        AudioControlManager audioControlManager,
+        OnAudioStartedUseCase onAudioStartedUseCase
     ) {
         audioControlManager.init(onAudioStartedUseCase);
     }
+
+    public static void destroyControllers() {
+        getControllerFactory().destroyControllers();
+    }
+
+    public static void reset() {
```

Comment:
It seems to me it makes sense to be more specific in naming because it might be difficult to get an idea how is `destroyControllers()` different from `reset()`. Maybe it's better to rename to `destroyControllersAndResetEngagement()`. Long name, I know. But it's clear.

Alternatively, we can add parameter `shouldResetEngagement` to `destroyControllers()` and have one function instead of two.

Reply:
I like the idea, but I'm also considering unifying these methods with the `ReleaseResourcesUseCase`. It's just an idea that I'm exploring at the moment, but it might be beneficial to _**release**_ all resources from a single location.
What do you think?

Reply:
Yes, I think it's a good idea to release all resources from a single place.